db/migrate/20190607014201_add_column_approved_to_institutions.rb

```diff
@@ -1,5 +1,5 @@
 class AddColumnApprovedToInstitutions < ActiveRecord::Migration
   def change
-    add_column :institutions, :approved, :boolean, null: false, default: false
```

Comment:
@dmccurdybah -  Instead of making changes to an existing migration, perhaps create a new one and use the [`change_column`](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column) method.  This will not impact existing data,  only data moving forward (post-migratation).  This should also help to mitigate against any migration edge cases.

Can you also pls explain the need for removing the defaults, or link to where this conversation is going on, for context?

cc: @jholton

Reply:
This is because of a performance issue when running this migration in prod.  @jholton manually ran his own and will be opening a PR, so I'm closing this one.